cli/cmd/promlensPortForward.go

```diff
@@ -0,0 +1,82 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+const LISTEN_PORT_PROMLENS = 8081
+const FORWARD_PORT_PROMLENS = 8080
+const LISTEN_PORT_CONNECTOR = 9201
+const FORWARD_PORT_CONNECTOR = 9201
```

Comment:
Isn't a single const enough? Reminds me about tslint standards.

Reply:
Logically they are different so I think better to keep separate